app/views/users/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <section class="dashboard-main">
   <h1> <%= current_user.first_name %>'s Dashboard </h1>
-
+  <%= link_to "Connect to Github", github_login_path %>
```

Comment:
Add link to connect to github that will need to be made into a button. We started with a link because that is what we were familiar with.